File: Spartane.Core/Domain/Generador_Folio_Encuesta/Generador_Folio_Encuesta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Spartane.Core.Domain.Modulo_Encuesta;
using Spartane.Core.Domain.Region;

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Web.Mvc;

namespace Spartane.Core.Domain.Generador_Folio_Encuesta
{
    /// <summary>
    /// Generador_Folio_Encuesta table
    /// </summary>
    public class Generador_Folio_Encuesta: BaseEntity
    {
        public int Folio { get; set; }
        public DateTime? Fecha_Genera { get; set; }
        public short? Anio_Genera { get; set; }
        public int? Modulo_Encuesta { get; set; }
        public int? Region { get; set; }
        public int? Consecutivo { get; set; }
        public string Folio_Encuesta_Ultimo_Generado { get; set; }

        [ForeignKey("Modulo_Encuesta")]
        public virtual Spartane.Core.Domain.Modulo_Encuesta.Modulo_Encuesta Modulo_Encuesta_Modulo_Encuesta { get; set; }
        [ForeignKey("Region")]
        public virtual Spartane.Core.Domain.Region.Region Region_Region { get; set; }

    }
	
	public class Generador_Folio_Encuesta_Datos_Generales
    {
                public int Folio { get; set; }
        public DateTime? Fecha_Genera { get; set; }
        public short? Anio_Genera { get; set; }
        public int? Modulo_Encuesta { get; set; }
        public int? Region { get; set; }
        public int? Consecutivo { get; set; }
        public string Folio_Encuesta_Ultimo_Generado { get; set; }

		        [ForeignKey("Modulo_Encuesta")]
        public virtual Spartane.Core.Domain.Modulo_Encuesta.Modulo_Encuesta Modulo_Encuesta_Modulo_Encuesta { get; set; }
        [ForeignKey("Region")]
        public virtual Spartane.Core.Domain.Region.Region Region_Region { get; set; }

    }


}

File: Spartane.Web/Uploads/Scripts/Generador_Folio_Encuesta.js
$(document).ready(function () {
    var inpuElementArray = [
        { "inputId": "Folio", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Fecha_Genera", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Anio_Genera", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Modulo_Encuesta", "inputType": "select", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Region", "inputType": "select", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Consecutivo", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Folio_Encuesta_Ultimo_Generado", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }

    ];
    setInputEntityAttributes(inpuElementArray, "#", 1);
});


